tests/webhook_test.go

```diff
@@ -157,12 +158,21 @@ var _ = Describe("Validation webhook", func() {
 			strategy.RevertToOriginalSspCr()
 		})
 
-		It("[test_id:6057] should fail to update commonTemplates.namespace", func() {
-			originalNs := foundSsp.Spec.CommonTemplates.Namespace
-			foundSsp.Spec.CommonTemplates.Namespace = originalNs + "-updated"
+		It("[test_id:6057] should not fail to update commonTemplates.namespace", func() {
```

Comment:
You could remove this test, because the webhook does not restrict the namespace anymore. 
Or if you want, you can add a new test to `tests/commonTemplates_test.go` to check if templates are deployed to different namespace when the `.spec.commonTemplates.namespace` is changed.